build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -24,10 +23,12 @@
     <PackageReference Include="WindowsAzure.Storage" Version="7.2.1" />
     <PackageReference Include="NuGet.CommandLine.XPlat" Version="$(CLI_NuGet_Version)" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(CLI_MSBuild_Version)" />
+    <PackageReference Include="Microsoft.Build.Runtime" Version="$(CLI_MSBuild_Version)" />
+
     <!-- This dependency was added due to an issue in restore where a lower version of this package coming from nuget.commandline.xplat
     led to an error. This is tracked as NuGet issue : https://github.com/NuGet/Home/issues/4213 -->
     <PackageReference Include="Microsoft.Build.Framework" Version="$(CLI_MSBuild_Version)" />
-    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(PlatformAbstractionsVersion)" />
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.0.0" />
```

Comment:
Do we know why we are pinning this?

Reply:
It blocked the ingestion of core-setup 2.1. Basically, build projectscan't run on stage0 2.0 if they ref 2.1. There is no need for the build projects to reference the version the product will build against. I think we should pin all of the versions here to something stable unless we actually require a new feature in a build project.

Reply:
We could put it back now but then we'll hit the same bug on the next core-setup version bump.